frontend/packages/dev-console/src/components/topology/shapes/BaseNode.tsx

```diff
@@ -81,37 +83,39 @@ export default class BaseNode extends React.Component<BaseNodeProps, State> {
             r={outerRadius}
             filter={hover ? createSvgIdUrl(FILTER_ID_HOVER) : createSvgIdUrl(FILTER_ID)}
           />
-          <image
-            x={-innerRadius}
-            y={-innerRadius}
-            width={innerRadius * 2}
-            height={innerRadius * 2}
-            xlinkHref={
-              getImageForIconClass(`icon-${icon}`) || getImageForIconClass('icon-openshift')
-            }
-          />
-          {label != null && (
-            <SvgBoxedText
-              className="odc-base-node__label"
-              y={outerRadius + 20}
-              x={0}
-              paddingX={8}
-              paddingY={4}
-              kind={kind}
-            >
-              {selected || hover ? label : truncateEnd(label)}
-            </SvgBoxedText>
-          )}
-          {selected && (
-            <circle
-              className="odc-base-node__selection"
-              cx={0}
-              cy={0}
-              r={outerRadius + 1}
-              strokeWidth={2}
+          <g className={`odc-base-node__contents${isDragging ? ' is-highlight' : ''}`}>
```

Comment:
We typically want to use the `classnames` library for joining classes.
This works fine though too.